db/models/product.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const Sequelize = require('sequelize');
+const db = require('APP/db');
+
+const productCategories = ['beauty', 'entertainment', 'fashion', 'food', 'sports', 'technology'].sort();
+
+const Product = db.define('products', {
+  name: {
+    type: Sequelize.STRING,
+    allowNull: false,
+    validate: {
+      notEmpty: true
+    }
+  },
+  category: {
+    type: Sequelize.ENUM(...productCategories),
+    allowNull: false,
+    validate: {
+      notEmpty: true
+    }
+  },
+  price: {
+    type: Sequelize.INTEGER,
```

Comment:
Do we want to maybe use

```
Sequelize.DECIMAL
```

*if we decide that our selling prices do not necessarily need to be whole numbers.